.github/workflows/pkgdown.yaml

```diff
@@ -0,0 +1,64 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [prod]
+  pull_request:
+    branches: [main, prod]
+  release:
+    types: [published]
+  workflow_dispatch:
+
+name: pkgdown
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # Only restrict concurrency for non-PR jobs
+    concurrency:
+      group: pkgdown-${{ github.event_name != 'pull_request' || github.run_id }}
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+      id-token: write
+      pages: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-pandoc@v2
+        with:
+          pandoc-version: "2.19.2"
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: "release"
+          use-public-rspm: true
+          install-r: false
+          extra-repositories: "https://mc-stan.org/r-packages/"
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          pak-version: rc
+          extra-packages: any::pkgdown local::.
+          needs: website
+
+      - name: Build site
+        run: "pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)"
+        shell: Rscript {0}
+
```

Comment:
Need to upload the artifact at the end of this. Compare 

https://github.com/CDCgov/wastewater-informed-covid-forecasting/blob/ce70b29453f46cac44b1100f6f46d11624db2715/.github/workflows/pkgdown.yaml#L55-L58

Reply:
Ok I think I am missing something here, but I am getting the error:
```
tar: wwinference/docs/: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
```

In the example from the other ww repo, we don't have `docs` folder within the repo either? So I am not sure what is missing here, I guess I thought this would somehow be generated with what is called above...

Reply:
Yes, it will be generated dynamically. Here it will just be called `docs/` because there isn't a subdirectory.

Reply:
You can try running 

```bash
Rscript "pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)"
```

at your local command line from the top-level project directory. A `docs/` folder should be generated.